packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -2197,6 +2198,45 @@ QUnit.module('aria accessibility', () => {
         assert.strictEqual($input.attr('role'), 'combobox', 'role attribute should remain assigned to the combobox after deleting');
     });
 
+    [
```

Comment:
Let's try to simplify it, for example, as follows. It's difficult to understand the mapping in the test title and inside with expectedId

```
[
        { label: 'Label' },
        { label: 'Label', inputAttr: { 'aria-label': 'Label' } },
        { label: undefined },
        { label: '' },
    ].forEach((options) => {
        QUnit.test(`component with fieldTemplate should have proper aria-labelledby attribute, options: ${JSON.stringify(options)} (T1230635)`, function(assert) {
            const $dropDownEditor = $('#dropDownEditorSecond').dxDropDownEditor({
                dataSource: ['one', 'two', 'three'],
                fieldTemplate: (data) => {
                    return $('<div>').dxTextBox({ value: data });
                },
                valueChangeEvent: 'keyup',
                ...options
            });
            let $input = $dropDownEditor.find(`.${TEXT_EDITOR_INPUT_CLASS}`);
            const $label = $dropDownEditor.find(`.${TEXT_EDITOR_LABEL_CLASS}`);

            const expectedAriaLabelledByValue = !options.inputAttr && options.label ?  $label.attr('id') : undefined;

            assert.strictEqual($input.attr('aria-labelledby'), expectedAriaLabelledByValue, `aria-labelledby attribute value after initialization`);

            keyboardMock($input)
                .type('a');

            $input = $dropDownEditor.find(`.${TEXT_EDITOR_INPUT_CLASS}`);

            assert.strictEqual($input.attr('aria-labelledby'), expectedAriaLabelledByValue, `aria-labelledby attribute value after typing`);

            keyboardMock($input)
                .caret(1)
                .press('backspace');

            $input = $dropDownEditor.find(`.${TEXT_EDITOR_INPUT_CLASS}`);

            assert.strictEqual($input.attr('aria-labelledby'), expectedAriaLabelledByValue, `aria-labelledby attribute value after deleting`);
        });
    })
```